en/ros/offboard_control.md

```diff
@@ -14,7 +14,7 @@ There are two things you want to setup on the firmware side before starting offb
 
 ### Map an RC switch to offboard mode activation
 
-To do this, load up the parameters in *QGroundControl* and look for the RC_MAP_OFFB_SW parameter to which you can assign the RC channel you want to use to activate offboard mode.
+In _QGroundControl_ you can set the [RC_MAP_OFFB_SW](../advanced_config/parameter_reference.md#RC_MAP_OFFB_SW) parameter to the RC channel that will be used to activate offboard mode.
 It can be useful to map things in such a way that when you fall out of offboard mode you go into position control.
```

Comment:
@beniaminopozzan This doesn't make much sense - why does this help you go into position control when you fall out of offboard mode? All it does is say that you can enable/disable offboard mode?
Or are you saying this is an easy way to disable offboard mode if needed, and by default you'll fall into position mode?

Reply:
@hamishwillee , it is indeed a bit misleading.
The RC switch allows you to enter and exit offboard mode, when lo leave it you'll return to the mode set by the mode switch in the RC which is not necessary position mode.
So line 18 should be "It can be useful to map things in such a way that when you fall out of offboard mode you go into **manual** control."
An even better approach is to use https://docs.px4.io/main/en/advanced_config/parameter_reference.html#COM_RC_OVERRIDE so the user just need to move the sticks to exit offboad though.

Reply:
Thank you! See https://github.com/PX4/PX4-user_guide/pull/3372